R/lmer_multimember.R

```diff
@@ -15,13 +16,20 @@
 #' )
 #' weights <- weights_from_vector(df$memberships)
 #' lmer(y ~ x + (1|members), df, memberships=list(members=weights))
-lmer <- function(formula, data, memberships=NULL, ...) {
+lmer <- function(formula, data = NULL, REML = TRUE, control = lme4::lmerControl(),
+                 start = NULL, verbose = 0L, weights = NULL, na.action=na.omit,
+                 offset = NULL, contrasts = NULL, devFunOnly = FALSE, memberships=NULL) {
+  ## TODO: call weights_from_vector from within this function
```

Comment:
I think we shouldn't call `weights_from_vector` from inside `lmer`. It's more flexible if `lmer` takes a membership/weight matrix as an argument and we leave it up to the user how they generate that matrix. (But of course we'll provide helper functions like `weights_from_vector`.)

Reply:
okidoki, less work that way :smile: